compose.js

```diff
@@ -88,12 +88,12 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('configuration', assign);
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
-    if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    srcDescriptor.initializers.forEach(i => {
-      if (isFunction(i) && dstDescriptor.initializers.indexOf(i) < 0) {
-        dstDescriptor.initializers.push(i);
+    dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
```

Comment:
Well I hope you have tested it little, because I did not, I just wrote it from top of my head.